dataland-frontend/src/components/forms/parts/fields/YesNoFormField.vue

```diff
@@ -59,10 +139,22 @@ export default defineComponent({
         No: "No",
       },
       isMounted: false,
+      qualityOptions: Object.values(QualityOptions).map((qualityOption: string) => ({
+        label: qualityOption,
+        value: qualityOption,
+      })),
+      currentValue: undefined,
+      currentReportValue: "",
+      currentPageValue: "",
+      currentQualityValue: "",
```

Comment:
I think these three are not required for this component to work. (Similar as in DataPointFormField.vue)

Reply:
👍